shared_workflow/load_config.py

```diff
@@ -3,7 +3,7 @@
 
 # Shouldn't this be in shared or qcore?
 def load(
-    directory=os.path.dirname(os.path.abspath(__file__)),
+    directory=os.path.join(os.path.dirname(os.path.abspath(__file__)), "../scripts"),
```

Comment:
Use `os.path.abspath(os.path.join(os.path.dirname(__file__), "../scripts"))` to get the same effect, but resolve the `..`?